react/client/src/styles/GlobalStyle.ts

```diff
@@ -2,39 +2,51 @@ import styled, { createGlobalStyle } from 'styled-components';
 import device from './breakpoints';
 
 export const GlobalStyle = createGlobalStyle`
-* {
-  margin:0;
-  pardding:0;
-  box-sizing: border-box;
-}
+  body {
+    background: #9903ff;
+    font-family: Roboto;
+    font-weight: 400;
+    height: 100vh;
+  }
 
-body {
-  background: #9903ff;
-  font-family: Roboto;
-  font-weight: 400;
-}
+  #root {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+  }
```

Comment:
Fixin and adjustin some styles so you can have it all look nice! You can see that I removed a lot of the special purpose `.ts` files in `/styles` since they were a bit overkill.